homeassistant/components/withings/common.py

```diff
@@ -465,8 +466,43 @@ class StateData:
 
 
 WITHINGS_MEASURE_TYPE_MAP: dict[
```

Comment:
I don't remember the reason for this map. Is it to get a homogeneous type (Measurement) after the lookup? The names of the enums on both sides seem to be a 1:1 map.

Reply:
I'm not 100% sure either, but it is blocking the split of the entity descriptions.
Maybe it can be dropped at a later stage...